chart/templates/mayastor/agents/core/agent-core-deployment.yaml

```diff
@@ -50,6 +50,7 @@ spec:
             - "--pool-commitment={{ .Values.agents.core.capacity.thin.poolCommitment }}"
             - "--volume-commitment-initial={{ .Values.agents.core.capacity.thin.volumeCommitmentInitial }}"
             - "--volume-commitment={{ .Values.agents.core.capacity.thin.volumeCommitment }}"
+            - "--faulted-child-wait-period={{ .Values.agents.core.faultedChildWaitPeriod }}"
```

Comment:
```suggestion
            - "--volume-commitment={{ .Values.agents.core.capacity.thin.volumeCommitment }}"{{ if .Values.agents.core.faultedChildWaitPeriod }}
            - "--faulted-child-wait-period={{ .Values.agents.core.faultedChildWaitPeriod }}"{{ end }}
```

Reply:
> Should we rename faultedChildWaitPeriod to something else, wait period might not be very meaningful unless you're familiar with the implementation?
Example: partialRebuildTimeout... something like that
@avishnu wdyt?

This was discussed here:
https://github.com/openebs/mayastor-extensions/pull/204#discussion_r1172122374

Reply:
My preference would be to keep the helm field name similar to the code variable name. Isn't it the norm?